### PR TITLE
Readme Windows info, 2023 update

### PR DESCRIPTION
**Pull request by CameliaMallus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,4 +52,4 @@ gunicorn app:app -w 2 --threads 3
 
 Founders of this project : [CameliaMallus](https://github.com/CameliaMallus) and [Marty42780](https://github.com/Marty42780).
 
-2022
+2023
```
Comment (Marty42780) [inline README.md]: I propose :
```
2022-2023
CC BY
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,13 @@
 ```bash
 apt install -y libcairo2-dev
 ```
+(On Windows, put pipwin directory in %PATH%)
+```
+pip install pipwin
+
+pipwin install cairocffi
```
Comment (Marty42780) [inline README.md:24]: [Follow the instruction here](https://cairographics.org/download/)

**Review by Marty42780 — CHANGES_REQUESTED** [review]
Good job, over-review appreciated

**Comment by CameliaMallus** [comment]
Committed suggested changes.

**Review by Marty42780 — APPROVED** [review]
Good project improvements

**Pull request merged**